strawberryfield.routing.yml

```diff
@@ -26,4 +26,16 @@ strawberryfield.important_solr_settings_form:
   requirements:
     _permission: 'access administration pages'
   options:
-    _admin_route: TRUE
\ No newline at end of file
+    _admin_route: TRUE
+    
+strawberryfield.custom_node_edit: 
+  path: '/node/{node}/custom-edit'
+  defaults:
+    _entity_form: 'node.edit'
+  requirements:
+    _entity_access: 'node.update'
+    _permission: 'access content'
```

Comment:
Ok, so what is missing here is the permission callback that checks if the user has access to any Form Mode other than Default.